### PR TITLE
feat: cart Items Quantity in SfHeader

### PR DESCRIPTION
**Pull request by aniamusial**
# Related issue
#654 

# Scope of work
1. Created badge 
2. Updated stories

# Checklist

- [x] I followed [composition rules](https://docs.storefrontui.io/contributing/coding-guidelines.html#component-rules) for my component
- [x] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

**Comment by pspaczek** [comment]
@aniamusial please check SfCircleIcon :) it has badge 👍 can you check it and try to adjust this issues 💚   
```vue
<slot v-if="hasBadge" name="badge">
  <div class="sf-circle-icon__badge"></div>
</slot>
```

File: packages/vue/src/components/organisms/SfHeader/SfHeader.vue [hunk 1]
```diff
@@ -52,6 +52,9 @@
           :aria-pressed="activeIcon === 'cart' ? 'true' : 'false'"
           @click="$emit('click:cart')"
         />
+        <div v-if="isCartEmpty" class="sf-header__icon--has-items">
```
Comment (pspaczek) [inline packages/vue/src/components/organisms/SfHeader/SfHeader.vue]: outside cart_icon is not a good idea ;)

**Comment by aniamusial** [comment]
Moved to #734 as way too many merge conflicts to be resolved

**Comment by pspaczek** [comment]
sorry @aniamusial  😞

**Pull request closed without merging**